easydiffraction/Profiles/P1D.py

```diff
@@ -257,6 +300,12 @@ class Instrument1DCWParameters(BaseObj):
     reflex_asymmetry_p2: ClassVar[Parameter]
     reflex_asymmetry_p3: ClassVar[Parameter]
     reflex_asymmetry_p4: ClassVar[Parameter]
+    twotheta_range_min: ClassVar[Descriptor]
```

Comment:
Since we now have the assignable `Parameter` with `enabled = False`, why not just use it instead of making these `Descriptor`s?

Reply:
I would leave `Descriptor` here as a reminder to discuss this issue. I think we need to have `twotheta_range_min` as a `Descriptor` rather than a `Parameter`, and perhaps reconsider the importance of the `enabled` property for the `Parameter` class.